src/services/zapper.ts

```diff
@@ -275,7 +275,7 @@ export class ZapperService extends Service {
       sellToken = ZeroAddress;
     }
 
-    const url = `https://api.zapper.fi/v1/zap-in/vault/${zapProtocol}/transaction`;
+    const url = `https://api.zapper.fi/v2/zap-in/vault/${zapProtocol}/transaction`;
```

Comment:
The result has changed, refactored with https://github.com/yearn/yearn-sdk/pull/284/commits/acc8eaa40bd605a023f370a4f3860b0a06a5d4ea